src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -744,18 +742,20 @@ BOOST_AUTO_TEST_CASE(CReLUFunctionTest)
   crelu.Backward(desiredActivations, error, derivatives);
   for (size_t i = 0; i < activations.n_elem; ++i)
   {
-    BOOST_REQUIRE_CLOSE(activations.at(i), desiredActivations.at(i), 1e-3);
+    REQUIRE(activations.at(i) ==
+        Approx(desiredActivations.at(i)).epsilon(1e-3));
   }
   for (size_t i = 0; i < derivatives.n_elem; ++i)
   {
-    BOOST_REQUIRE_CLOSE(derivatives.at(i), desiredDerivatives.at(i), 1e-3);
+    REQUIRE(derivatives.at(i) ==
+        Approx(desiredDerivatives.at(i)).epsilon(1e-3));
```

Comment:
```suggestion
    REQUIRE(derivatives.at(i) ==
        Approx(desiredDerivatives.at(i)).epsilon(1e-3 / 100));
```

Reply:
Wouldn't you just use 1e-6 in that case?

Reply:
Yes that would work too

Reply:
Yes, my bad. 1e-5 :laughing: